libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialogBase/ShareDialogBase.tsx

```diff
@@ -52,6 +52,12 @@ export const ShareDialogBase: React.FC<IShareDialogBaseProps> = (props) => {
         onGranularGranteeShareChange,
     } = useShareDialogBase(props);
 
+    const previouslyFocusedRef = useRef<HTMLElement>(null);
+
+    useEffect(() => {
+        previouslyFocusedRef.current = document.activeElement as HTMLElement;
```

Comment:
activeElement is already rendered on screen so you can use it directly in useRef initialization. I did the same issue in my code too